doc/dialects/aurora.md

```diff
@@ -62,7 +62,7 @@ The SQL statement below creates the adapter script, defines the Java class that
 ```sql
 CREATE OR REPLACE JAVA ADAPTER SCRIPT ADAPTER.JDBC_ADAPTER AS
     %scriptclass com.exasol.adapter.RequestDispatcher;
-    %jar /buckets/<BFS service>/<bucket>/virtual-schema-dist-8.0.0-bundle-6.0.0.jar;
+    %jar /buckets/<BFS service>/<bucket>/virtual-schema-dist-8.0.0-posgresql-1.0.0.jar;
```

Comment:
I think this is a bad idea to specify versions here. We won't update this automatically anymore. Let's use placeholders